presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+* Introduced BigQuery Connector.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve ORC writer performance for string columns.
+* Add support for improving cache affinity hit rate by preventing repartitioning when nodes go down for a limited time. This can be enabled by setting the `internal-communication.memoize-dead-nodes-enabled` configuration property
+* Add support to enforce NOT NULL constraint by checking values for NOT NULL columns.
+
+
+JDBC Connector Changes
+______________________
+* Add support to accept username and password.
+
+Big Query Connector Changes
+___________________________
+* Introduced Big Query Connector. See :doc:`/connector/bigquery` to get started
+
+Hive Changes
+____________
+* Support cache quota when using Alluxio cache
```

Comment:
`Add support for cache quota when using Alluxio cache`

Reply:
Fixed in latest revision